BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/Onboarding/DigitalInvoiceOnboardingViewController.swift

```diff
@@ -93,7 +93,7 @@ final class DigitalInvoiceOnboardingViewController: UIViewController {
         firstLabel.adjustsFontForContentSizeCategory = true
 
         secondLabel.text = secondLabelText
-        secondLabel.font = configuration.textStyleFonts[.headline]
```

Comment:
[clarification] I checked our Figma documentation for this screen and the right font is `headline`. BUT, for the normal onboardings, we mentioned `subheadline`. Do we want to change the Figma documentation file after this?

Reply:
Yes we had to do that.
It will be a `headline` on the both Onboarding screens.